.github/workflows/dependency-check.yml

```
@@ -0,0 +1,46 @@
+name: dependency check
```

Let's rename this file to be more detailed
```suggestion
name: cve dependency scan
description: scan runtime libraries for CVE issues and report to slack if found
```